test/views/templates/payments/PaymentsHistoryChargeDescriptionTemplateSpec.scala

```diff
@@ -2360,6 +2361,62 @@ class PaymentsHistoryChargeDescriptionTemplateSpec extends ViewBaseSpec {
         elementText(Selectors.description) shouldBe "for period 12\u00a0Jan to 23\u00a0Mar\u00a02018"
       }
     }
+
```

Comment:
All Tests including non agent tests are in the `"user is an agent" when {` part. can the non agent tests be put with the rest of the tests to keep them organise

Reply:
Cool updated now